apps/postybirb-ui/src/components/form/fields/field-label.tsx

```diff
@@ -40,12 +40,19 @@ export function FieldLabel(
     <Input.Wrapper required={field.required} label={label}>
       {children}
       {errors.map((error) => (
-        <Input.Error key={`${propKey}-${error.id}`}>
+        <Input.Error
+          key={`${propKey}-${error.id}-${JSON.stringify(error.values)}`}
+          pb={4}
+        >
           <ValidationTranslation id={error.id} values={error.values} />
         </Input.Error>
       ))}
       {warnings.map((warning) => (
-        <Input.Error key={`${propKey}-${warning.id}`} c="orange">
+        <Input.Error
+          key={`${propKey}-${warning.id}-${JSON.stringify(warning.values)}`}
+          c="orange"
+          pb={4}
+        >
```

Comment:
Key is changed because of this
<img width="549" height="299" alt="Image" src="https://github.com/user-attachments/assets/798be50b-7792-4243-9c97-81c3b863e4b4" />

Before padding bottom:
<img width="373" height="124" alt="Image" src="https://github.com/user-attachments/assets/1547b7dd-d8de-4b7c-b391-cf3a42d53a69" />

After padding bottom:
<img width="358" height="140" alt="Image" src="https://github.com/user-attachments/assets/62c33cb7-0ed4-4a0f-af5f-d939165dde73" />